### PR TITLE
First six entities

### PR DESCRIPTION
**Pull request by NirmalManoj**

### CODE REVIEW

File: SocialMediaDB.sql [hunk 1]
```diff
@@ -1,3 +1,67 @@
...
+
+DROP TABLE IF EXISTS POST;
+CREATE TABLE POST (
+    post_id int not null AUTO_INCREMENT,
+    time TIMESTAMP not null,
+    text TEXT,
+    media varchar(10000),
+    PRIMARY KEY(post_id),
+    user_id int not null,
+    CONSTRAINT OWNS_POST FOREIGN KEY (user_id) REFERENCES USER (user_id),
```
Comment (alapan-sau) [inline SocialMediaDB.sql]: ```on DELETE CASCADE on UPDATE CASCADE```

File: SocialMediaDB.sql [hunk 2]
```diff
@@ -1,3 +1,67 @@
...
+
+
+DROP TABLE IF EXISTS COMMENT;
+CREATE TABLE COMMENT(
+    comment_id int not null AUTO_INCREMENT PRIMARY KEY,
+    time TIMESTAMP not NULL,
+    text TEXT,
+    media VARCHAR(10000),
+    user_id int not null,
+    CONSTRAINT OWNS_COMMENT FOREIGN KEY (user_id) REFERENCES USER (user_id),
```
Comment (alapan-sau) [inline SocialMediaDB.sql]: on DELETE CASCADE on UPDATE CASCADE

File: SocialMediaDB.sql [hunk 3]
```diff
@@ -1,3 +1,67 @@
...
+) ENGINE = InnoDB DEFAULT CHARSET = utf8;
+
+DROP TABLE IF EXISTS STORIES;
+CREATE TABLE STORIES(
+    story_id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    time TIMESTAMP,
+    text TEXT,
+    media VARCHAR(10000),
+    user_id int not null,
+    CONSTRAINT OWNS_STORY FOREIGN KEY (user_id) REFERENCES USER (user_id),
```
Comment (alapan-sau) [inline SocialMediaDB.sql]: ```on DELETE CASCADE on UPDATE CASCADE```

File: SocialMediaDB.sql [hunk 4]
```diff
@@ -1,3 +1,67 @@
...
+
+DROP TABLE IF EXISTS MESSAGE;
+CREATE TABLE MESSAGE(
+    message_id int not null PRIMARY KEY AUTO_INCREMENT,
+    text TEXT not null
+) ENGINE = InnoDB DEFAULT CHARSET = utf8;
+
+DROP TABLE IF EXISTS PROFILE;
+CREATE TABLE PROFILE (
+    user_id int PRIMARY KEY,
```
Comment (alapan-sau) [inline SocialMediaDB.sql:61]: ```on DELETE CASCADE on UPDATE CASCADE```

File: SocialMediaDB.sql (hunk 1 above)
Comment (NirmalManoj) [reply SocialMediaDB.sql]: Changed as suggested.

File: SocialMediaDB.sql (hunk 3 above)
Comment (NirmalManoj) [reply SocialMediaDB.sql]: Changed as suggested.

File: SocialMediaDB.sql (hunk 2 above)
Comment (NirmalManoj) [reply SocialMediaDB.sql]: Changed as suggested.

Comment (pavani-17) [reply SocialMediaDB.sql]: add post_id too ig

File: SocialMediaDB.sql [hunk 5]
```diff
@@ -1,3 +1,67 @@
...
+    date_of_birth DATE not null,
+    sex varchar(30),
+    sexes ENUM('Male', 'Female', 'Others', 'PreferNotToSay'),
+    education varchar(1000)
```
Comment (pavani-17) [inline SocialMediaDB.sql]: remove education, it is multi valued

File: SocialMediaDB.sql (hunk 2 above)
Comment (alapan-sau) [reply SocialMediaDB.sql]: Either remove the user_id or add the post_id

Comment (NirmalManoj) [reply SocialMediaDB.sql]: I have removed user_id

**Review by alapan-sau — COMMENTED** [review]
Done!

File: SocialMediaDB.sql (hunk 4 above)
Comment (alapan-sau) [reply SocialMediaDB.sql:61]: And it is a foreign key

Comment (NirmalManoj) [reply SocialMediaDB.sql:61]: Done.

**Review by pavani-17 — APPROVED** [review]
Everything fine!

**Review by alapan-sau — APPROVED** [review]

File: SocialMediaDB.sql (hunk 5 above)
Comment (NirmalManoj) [reply SocialMediaDB.sql]: I removed it already. Refresh.

**Pull request merged**